e2e/test/scenarios/embedding/embed-resource-downloads.cy.spec.ts

```diff
@@ -375,12 +378,18 @@ H.describeWithSnowplowEE(
             "leffler.dominique@hotmail.com",
             FILTER_VALUES[0],
           ];
+          const SECOND_ROW = [
```

Comment:
This strengthens the assertion for the existing test to ensure that both parameter values are used in the query. The rest of this test is the same.